Composer/packages/extensions/visual-designer/src/components/menus/EdgeMenu.tsx

```diff
@@ -5,13 +5,18 @@
 import { jsx, css } from '@emotion/core';
 import { useCallback, useContext, useState } from 'react';
 import classnames from 'classnames';
-import formatMessage from 'format-message';
-import { createStepMenu, DialogGroup, SDKKinds } from '@bfc/shared';
-import { IContextualMenu, ContextualMenuItemType } from 'office-ui-fabric-react/lib/ContextualMenu';
+import formatMessage, { custom } from 'format-message';
+import { createStepMenu, createStepSubmenu, DialogGroup, SDKKinds } from '@bfc/shared';
```

Comment:
This would be a good time to remove `createStepMenu` & `createStepSubmenu` since they are only used in the visual editor.

Reply:
@a-b-r-o-w-n I'd like to. I will move them out of shard package